releasenotes/README.md

```diff
@@ -16,6 +16,10 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
+* NEW: Some commands (`force:org:open`, `force:org:display`, `force:user:display`, `force:source:open`) output tokens and URLs with tokens embedded in them.  They now include a warning about the risks of sharing that sensitive information. 
+
+* NEW: `force:source:retrieve -m CustomField` (or the manifest equivalent using `*` for CustomField without CustomObject entries) succeeds but returns no results.  It's a confusing Metadata API quirk.  Now, when you run the command like that, it will output a warning with instructions for how to do what you probably meant to do.
```

Comment:
Can we use different wording for "It's a confusing Metadata API quirk"? ...no results due to an issue with how the Metadata API interprets the command flag (I don't know what the quirk is but something like that?)

Reply:
it's not really an "issue" (like a problem that'll be fixed) just more a thing people don't know that confuses them.

How about "It's confusing, but that's how the Metadata API works" 

Can you say what the goal of different wording would be?

Reply:
Just to reframe it so it doesn't sound like a criticism. If it's WAD albeit confusing, perhaps just say (It's just how Metadata API works but we want to provide you the results that you're looking for.) Something like that. It's totally a judgment call, of course.